org.eclipse.tm4e.registry/src/main/java/org/eclipse/tm4e/registry/internal/AbstractGrammarRegistryManager.java

```diff
@@ -122,7 +122,8 @@ public IGrammar getGrammarForFileType(String fileType) {
 	 * 

```

Comment:
Overall, this file doesn't have any payload change, it'd be better to just not change it at all.